setup.py

```diff
@@ -25,7 +25,7 @@
     install_requires=[
         'numpy',
         'onnxruntime',
-        'onnx==1.8.1',
+        'onnx>=1.7.0,<=1.11.0',
```

Comment:
I changed the minimum here to 1.7.0 to match supported_onnx.json, which allowed down to version 1.7.0.  Let me know if this is backwards, and I should instead update supported_onnx.json to have a minimum version of 1.8.1.